wiki/Manual.md

```diff
@@ -4876,6 +4876,10 @@ To cancel an existing order use
 - `cancelAllOrders ()` for all open orders
 - `cancelAllOrdersAfter ()` for all open orders after the given timeout
 
+Note: typically batch order-canceling methods return an array of orders, where orders statuses would be noted e.g. `status: success/rejected/etc`. However, there might also be cases, for example, if only one order was not cancelled and exchange responded with `{success:false, reason:"order id 123 does not exist"}`. In such cases, CCXT might also throw an exception instead of returning an array.
```

Comment:
@ttodua this might not be correct, sometimes we try to parse these errors inside cancelOrders and still return an array

Reply:
but it doesn't contradict to what I noted there, because i say that "typically we do that" but can you guarantee taht we do that for all 100+ exchanges? i left a room for "possible different expectations" while noting our unified approach

Reply:
what I am about is that we havent (and can't atm) check all exchanges to say guaranteely that those errors are not picked up in handleErrors and are being parsed in cancelOrders for all exchanges. so, on those exceptionary exchanges we might be missing such parsing, so let users know that "in some cases exception might be thrown"

Reply:
BUt I still think this might be confusing, because in some cases the exchange throws an exception when 1 of the orders fails, but we still muffle it and the users might assume that if the exchange throws the error the error will bubble up to the user land

Reply:
But how to tell that in other words? please feel free to make an edit